activiti-cloud-full-example/values.yaml

```diff
@@ -5,10 +5,13 @@
 
 global:
   keycloak:
-    url: "http://activiti-keycloak.REPLACEME/auth"
+    url: &keycloakhost "http://activiti-keycloak.REPLACEME/auth"
```

Comment:
Use of value references limits the reuse of the chart with Helm shell, CI/CD, and value.yaml overrides. The references work only within the defined scope. If someone provides values.yaml or uses Helm install with --set option to customize keycloak url value, this chart configuration will break and install wrong values in origin and Ingress hosts.